app/views/users/_edit_form.html.erb

```diff
@@ -19,14 +19,14 @@
 
       <div class="form-group col-sm-6">
         <label for="username"><%= t('users._form.username') %></label>
-        <%= f.text_field :username, { readonly: true, tabindex: 1, placeholder: "Username", class: 'form-control', id: 'username' , title: 'Username field cannot be changed', disabled: true} %>
+        <%= f.text_field :username, { readonly: true, tabindex: 1, placeholder: "Username", class: 'form-control', id: 'username' , name: 'username', title: 'Username field cannot be changed', disabled: true} %>
         <input type="hidden" name="id" value="<%= @user.username %>" />
       </div>
 
       <div class="form-group col-sm-6">
 
         <label for="email"><%= t('users._form.email') %></label>
-        <%= f.text_field :email, { tabindex: 3, required: true, pattern: '[^@]+@[^@]+\.[a-zA-Z]{2,6}', placeholder: "you@email.com", class: 'form-control', id: 'email' } %>
```

Comment:
Hi @gauravano 
I removed this validation of email using as this has many failed cases.
eg - 
* It doesn't allow lekhika@lekhika which is considered valid email.
https://stackoverflow.com/questions/20573488/why-does-html5-form-validation-allow-emails-without-a-dot
* It doesn't allow the lekhika@hotmail.engineer because the regex limited the length of domain name to max 6 characters which isn't correct..
* Also, I've used jquery validation library which automatically performs reatlime email validation.

Reply:
We can remove but please check if rails allow it or not.